components/notifier.rst

```diff
@@ -0,0 +1,174 @@
+.. index::
+   single: Notifier
+   single: Components; Notifier
+
+The Notifier Component
+====================
+
+    The Notifier component helps sending emails.
+
+If you're using the Symfony Framework, read the
+:doc:`Symfony Framework Notifier documentation </notifier>`.
+
+Installation
+------------
+
+.. code-block:: terminal
+
+    $ composer require symfony/notifier
+
+.. include:: /components/require_autoload.rst.inc
+
+Email
+-----
+
+The Notifier component has notify you when something goes wrong::
+
+    use Symfony\Bridge\Twig\Mime\NotificationEmail;
```

Comment:
generaly the doc use to put such code in context, inside a controller or a service for example
can you reproduce the same? i think it will be very usefull :)